types_test.go

```diff
@@ -163,8 +189,10 @@ func TestConversion(t *testing.T) {
 	c.Cfg2 = &ConvConfig{"xx", 2}
 	c.Cfg3 = &ConvConfig{"zz", 3}
 	c.Slice = []*ConvConfig{{"yy", 4}, {"ff", 5}}
+	c.Nullable1 = &Nullable{Data: "test"}
+	c.Nullable2 = nil
 
-	_, err := testEngine.Insert(c)
+	_, err := testEngine.Nullable("nullable2").Insert(c)
```

Comment:
I don't like the idea with `session.Statement.nullableMap`. Looks like it can be filled only from `.Sync(c)`. I`m not sure is all use this method for migration.